frameworks/cassandra/src/main/dist/cassandra.yaml

```diff
@@ -111,8 +111,8 @@ snapshot_before_compaction: false
 auto_snapshot: {{CASSANDRA_AUTO_SNAPSHOT}}
 cross_node_timeout: false
 endpoint_snitch: GossipingPropertyFileSnitch
-roles_update_interval_in_ms: {{CASSANDRA_ROLES_UPDATE_INTERVAL_IN_MS}}
-permissions_update_interval_in_ms: {{CASSANDRA_PERMISSIONS_UPDATE_INTERVAL_IN_MS}}
```

Comment:
what is the reason for `CASSANDRA_` prefix being dropped for the new/updated parameters while is present for the rest of config options?

Reply:
as variables are moved from Cassandra tab to service.security under config.json, so i changed marathon variable name as well.